### PR TITLE
Fix(audit-logs): improved audit log to have old project slug

### PR DESCRIPTION
**Pull request by maxiuyuan**
Added the old project slug in the PROJECT_EDIT message in audit log.

before: 
image
after:
image

JIRA:https://getsentry.atlassian.net/browse/ER-731

### CODE REVIEW

**Review by leedongwei — COMMENTED** [review]
Screenshots might be wrong, the "before" is showing new/old slugs too?

Would it be better to have an entirely different string for renaming slugs? So something like

```
if self.data["old_slug"] is not None:
    return "renamed project slug from " + self.data["old_slug"] + " to " + self.data["new_slug"]

return "edited project settings " + (" ".join(f" in {key} to {value}" for (key, value) in self.data.items())
```

File: tests/sentry/api/endpoints/test_project_details.py [hunk 1]
```diff
@@ -118,6 +118,12 @@ def test_project_renamed_302(self):
         )
 
         response = self.get_valid_response(project.organization.slug, project.slug, status_code=302)
+        assert (
+            AuditLogEntry.objects.get(
+                organization=project.organization, event=AuditLogEntryEvent.PROJECT_EDIT
+            ).data["old_slug"]
+            == project.slug
```
Comment (JoshFerge) [inline tests/sentry/api/endpoints/test_project_details.py:125]: want to add an assertion for new slug too?

File: src/sentry/models/auditlogentry.py [hunk 2]
```diff
@@ -280,6 +280,8 @@ def get_note(self):
         elif self.event == AuditLogEntryEvent.PROJECT_EDIT:
             return "edited project settings " + (
                 " ".join(f" in {key} to {value}" for (key, value) in self.data.items())
+                if self.data["old_slug"] is None
+                else "in " + self.data["old_slug"] + " to " + self.data["new_slug"]
```
Comment (JoshFerge) [inline src/sentry/models/auditlogentry.py:284]: is it possible to edit multiple settings at once? does this need to be in the loop? also, should it say something to the effect of
>changed project setting "slug" from "old_slug" to "new_slug"

**Comment by maxiuyuan** [comment]
> Screenshots might be wrong, the "before" is showing new/old slugs too?
> 
> Would it be better to have an entirely different string for renaming slugs? So something like
> 
> ```
> if self.data["old_slug"] is not None:
>     return "renamed project slug from " + self.data["old_slug"] + " to " + self.data["new_slug"]
> 
> return "edited project settings " + (" ".join(f" in {key} to {value}" for (key, value) in self.data.items())
> ```

in the before, the "new_slug" is a constant since its the key for the new slug, rather than the old slug

**Comment by leedongwei** [comment]
> in the before, the "new_slug" is a constant since its the key for the new slug, rather than the old slug

Lol yea that's super confusing. This is a good change.

**Review by leedongwei — APPROVED** [review]
LGTM

**Pull request merged**